<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReversiGameWindow</class>
 <widget class="QMainWindow" name="ReversiGameWindow">
  <property name="windowTitle">
   <string>Reversi / Othello (8x8)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="statusLabel">
      <property name="text">
       <string>Turn: Black</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="scoreLabel">
      <property name="text">
       <string>Score  Black: 2   White: 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="resetButton">
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="boardContainer">
      <layout class="QGridLayout" name="boardGrid">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
